server/events/models/models.go

```diff
@@ -131,6 +131,11 @@ func NewRepo(vcsHostType VCSHostType, repoFullName string, cloneURL string, vcsU
 		return Repo{}, fmt.Errorf("invalid repo format %q, repo %q should not contain any /'s", repoFullName, owner)
 	}
 
+	azuredevopsVcsToken := ""
```

Comment:
Can you add a test for this in models_test.go?

Reply:
Looking at this